### DISCUSSION TITLE
캐릭터 쉐이더 변경

### DISCUSSION THREAD

**Discussion by yoonjoohwan** [naverz/zepeto-studio-kor#860] [5. Q&A]
캐릭터의 쉐이더나 머터리얼을 변경할수 있습니까?

**Comment by toddee9999** [comment]
이 부분은 여느 유니티 프로젝트와 같습니다. 생성된 캐릭터의 Skinned Mesh Renderer에 접근하시면 변경 가능합니다 :)

**Reply by yoonjoohwan** [reply]
이렇게 해보고있는데요 유니티에서는 shader 가 잘변하는데 모바일 qr 테스트를하면 안변합니다 ㅠㅠ
![image](https://user-images.githubusercontent.com/91707090/172774012-0825a296-11da-44ca-9f01-17c6fc4fc285.png)

**Reply by yoonjoohwan** [reply]
핑크색으로 바뀝니다 error shader라고 나오더라구요

**Reply by Zepeto-TechEvangelist** [reply]
안녕하세요, 제페토 본체앱 버전이 3.13.0인지 확인 부탁 드립니다.
해당 본체앱에서 파티클 효과가 포함된 외형 아이템이 깨지는 문제가 있어서
연관된 문제가 아닐까 생각됩니다.

**Reply by yoonjoohwan** [reply]
혹시 앱버전이 3.13.0 이면 어떻게 해야할까요?

**Reply by Zepeto-TechEvangelist** [reply]
핫픽스 준비 중입니다. 버전 업데이트 이후 해결 될 것입니다.

**Reply by yoonjoohwan** [reply]
네 감사합니다

**Comment by YSHFM** [comment]
![image](https://user-images.githubusercontent.com/106574541/172779100-3e52e7f9-fdac-49eb-824e-be8b4b9ceb7c.png)
바디에 적용해본 림라이트 쉐이더입니다 저는 정상적으로 동작됩니다.
![image](https://user-images.githubusercontent.com/106574541/172779167-1c5d22a4-ef4f-442b-9329-c526ef11c297.png)
예시코드입니다. 모바일에서도 적용되는 것을 확인했습니다.

**Reply by yoonjoohwan** [reply]
혹시 앱버전 확인이 가능하신가요?

**Reply by YSHFM** [reply]
3.13.0이고 안드로이드랑 ipadOS에서 둘 다 적용되는 것 확인했습니다.

**Reply by yoonjoohwan** [reply]
ShaderManager 통하셔서 inspector 에 직접 Shader 파일을 넣어주신건가요?

**Reply by YSHFM** [reply]
네 맞습니다

**Reply by YSHFM** [reply]
![image](https://user-images.githubusercontent.com/106574541/172786844-ace76ae8-93a8-46f5-9e48-b5f4815aab80.png)
해당 원소들을 가져옵니다.

**Reply by YSHFM** [reply]
바디에는 적용되겠으나 의상 같은 경우는 적용이 안되실 수 있을 거 같네요
